src/atoms/fieldset/Fieldset.stories.js

```diff
@@ -1,53 +1,32 @@
-import { select, text } from '@storybook/addon-knobs'
-
-import getClassKnobsConfig from '@utils/helpers/get-class-knobs-config.js'
-import selectorsConfig from './Fieldset.selectors.json'
-
 import AFieldset from './Fieldset.vue'
```

Comment:
for some reason I can't see fieldset in storybook 
![image](https://user-images.githubusercontent.com/49198312/129327160-117c3fcc-96c8-4c90-855a-03a8e3dabfa7.png)